components/ui/QuickNote.tsx

```
@@ -0,0 +1,176 @@
+            </button>
+          </div>
+
+          {/* Content */}
+          <div className="p-4">
+            <textarea
+              value={content}
+              onChange={(e) => setContent(e.target.value)}
+              placeholder="Start typing... (auto-saved locally)"
+              className="w-full h-64 bg-[#1A1410] text-[#E8DCC4] border border-[#8B7355] rounded-md p-3 focus:outline-none focus:ring-2 focus:ring-[#A08465] resize-none"
+              autoFocus
+            />
```

Biome (2.1.2)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)